service/src/test/java/de/adorsys/psd2/sandbox/xs2a/pis/PIS.feature

```diff
@@ -24,9 +30,37 @@ Feature: Payment Initiation Service
       | payment-product       | psu-id | password | sca-method | tan   | sca-status | code |
       | sepa-credit-transfers | PSU-1  | 12345    | SMS_OTP    | 54321 | finalised  | 200  |
 
+
+    ################################################################################################
```

Comment:
Suggestion: put this scenario below the Comment-Block for `Payment Initiation` instead of creating a new comment-block `periodic payment`

Reply:
done